app/views/static_pages/index.html.erb

```diff
@@ -17,7 +17,18 @@
 </div>
 
 <div id="tab-cuisines" style="display:none;">
+  <div class="col-xs-3">
   this will be the cuisines tab.
+  </div>
+  <div class="col-xs-3" class="cuisines">
```

Comment:
What happened to the .filter-box class? I thought you were going to use it for all 3 tabs; why do you need to create a new class .cuisines?